specification/containerregistry/data-plane/Azure.ContainerRegistry/stable/2021-07-01/containerregistry.json

```diff
@@ -1635,7 +1635,7 @@
         "timeasc"
       ],
       "x-ms-enum": {
```

Comment:
The "x-ms-enum" definition is client only for defining an enum. What goes over the wire to the service is the outer name in the definition. In this case, "TagOrderBy".

@tjprescott, please let me know if you disagree and think this is a breaking change for the service.

Reply:
Not a breaking change for the service, but it will be for the SDK. Are you doing this to eliminate "manual renaming", or has this not yet been released in an SDK?

Reply:
@tjprescott - we are making this change as a feedback of API review for GA readiness. So, yes deliberately taking this SDK breaking change to update code in the clients for the upcoming release.

Reply:
Also, can you help me merge the PR please. The errors look legit, but they are for older pre-existing stuff.

Reply:
@Mohit-Chakraborty I can't merge the PR with the CI failure. You may need to get the service team to fix these errors.

Reply:
Understood. Let me add folks from the service team.